sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/tests/Azure.Monitor.OpenTelemetry.AspNetCore.Tests/LiveMetrics/LiveMetricsRedirectPolicyTests.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading;
+using Azure.Core;
+using Azure.Core.Pipeline;
+using Azure.Core.TestFramework;
+using Azure.Monitor.OpenTelemetry.AspNetCore.Internals.LiveMetrics;
+using Xunit;
+
+namespace Azure.Monitor.OpenTelemetry.AspNetCore.Tests.LiveMetrics
+{
+    public class LiveMetricsRedirectPolicyTests
+    {
+        [Fact]
+        public void VerifyLiveMetricsRedirectPolicyReadsHeader()
+        {
+            var mockTransport = new MockTransport(new MockResponse(200).AddHeader("x-ms-qps-service-endpoint-redirect-v2", "http://new.host/Service.svc"));
```

Comment:
Would a 302 or 200 status code be sent for redirect requests?

Reply:
service returns with a 200.